src/alitra/frame_transform.py

```diff
@@ -22,32 +21,11 @@ class FrameTransform:
     :param translation: translation from frame (from_) to frame (to_), expressed in the (to_) frame.
     """
 
-    def __init__(
+    def  __init__(
         self,
-        translation: Translation,
-        from_: Literal["robot", "asset"],
-        to_: Literal["robot", "asset"],
-        euler: Euler = None,
-        quaternion: Quaternion = None,
+        transform: Transform
     ):
-        if euler is None and quaternion is None:
-            raise ValueError("Euler or quaternion must be set to describe the rotation")
-        elif euler and quaternion:
-            raise ValueError("Specify only one rotation, either euler or quaternion.")
-        elif euler:
-            try:
-                self.transform = Transform(
-                    translation=translation, euler=euler, from_=from_, to_=to_
-                )
-            except ValueError as e:
-                raise ValueError(e)
-        elif quaternion:
-            try:
-                self.transform = Transform(
-                    translation=translation, quaternion=quaternion, from_=from_, to_=to_
-                )
-            except ValueError as e:
-                raise ValueError(e)
+        self.transform=transform
```

Comment:
Later we might want to  have several transform objects in the Frame_transform class. 

In this case, calling Frame_transform with from_ and to_ should select the appropriate transform